mobile/lib/modules/album/views/create_album_page.dart

```diff
@@ -26,7 +32,8 @@ class CreateAlbumPage extends HookConsumerWidget {
     final isAlbumTitleTextFieldFocus = useState(false);
     final isAlbumTitleEmpty = useState(true);
     final selectedAssets =
-        ref.watch(assetSelectionProvider).selectedNewAssetsForAlbum;
+        ref.watch(assetSelectionProvider).selectedNewAssetsForAlbum
+        ..addAll(initialAssets ?? []);
```

Comment:
This doesn't work quite the way I intended it to. Got any suggestions for how to implement this idea?

Reply:
I think I've resolved it below.